.github/workflows/soroban-rpc.yml

```diff
@@ -0,0 +1,52 @@
+name: Soroban RPC
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+
+  integration:
+    name: Integration tests
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go: [1.19.1]
+    runs-on: ${{ matrix.os }}
+    env:
+      SOROBAN_RPC_INTEGRATION_TESTS_ENABLED: true
+      SOROBAN_RPC_INTEGRATION_TESTS_CAPTIVE_CORE_BIN: /usr/bin/stellar-core
+      PROTOCOL_19_CORE_DEBIAN_PKG_VERSION: 19.4.0-1075.39bee1a2b.focal
```

Comment:
should this be bumped to most recent soroban core version:

19.5.1-1111.eba1d3de9.focal~soroban

Reply:
yes, I'll update that.